tests/e2e/solclient/ocr2.go

```diff
@@ -353,10 +353,10 @@ func (m *OCRv2) proposeConfig(ocConfig contracts.OffChainAggregatorV2Config) err
 	}
 	instr = append(instr, ocr_2.NewProposePayeesInstruction(
 		m.Mint.PublicKey(),
-		payees,
 		m.Proposal.PublicKey(),
 		m.Owner.PublicKey()).Build(),
 	)
+	// TODO: how to set payees as remaining
```

Comment:
@skudasov Any idea here? I need to set `remainingAccounts`

A misc idl contains an extra field somehow: https://github.com/gagliardetto/anchor-go/blob/29a52c5980ddecb5a0f0444a27b4e669f39ea74f/idl/examples/misc.json#L418=